ush/config_defaults.yaml

```diff
@@ -1787,12 +1787,19 @@ task_run_fcst:
   # weather model].  If this is set to true, the the run_post task will
   # be deactivated.
   #
+  # ITASKS:
+  # Variable denoting the number of write tasks in the i direction in the
+  # current group.  Used for inline post 2d decomposition.  Setting this
+  # variable to a value greater than 1 will enable 2d decomposition.
+  # Default setting is 1.
+  #
```

Comment:
```suggestion
  # Note that 2D decomposition does not yet work with GNU compilers, so 
  # this value will be reset to 1 automatically when using GNU compilers (i.e., COMPILER: gnu).
  #
```

Reply:
I've also replaced the lowercase d with uppercase D in the 2D decomposition documentation for ITASKS above this line.